tests/unit/modules/zypper_test.py

```diff
@@ -59,6 +60,28 @@ class ZypperTestCase(TestCase):
     Test cases for salt.modules.zypper
     '''
 
+    def setUp(self):
+        self.new_repo_config = dict(
+            name='mock-repo-name',
+            url='http://repo.url/some/path'
+        )
+        self.zypper_patcher_config = {
+            '_get_configured_repos': Mock(
+                side_effect=[
+                    Mock(**{'sections.return_value': []}),
+                    Mock(
+                        **{
+                            'sections.return_value': [
+                                self.new_repo_config['name']
+                            ]
+                        }
+                    ),
+                ],
+            ),
```

Comment:
You love this. 😉 But can it be less LISP like, please?

![Booya](http://www.geeksaresexy.net/wp-content/uploads/2009/08/520fujicodescreenshot.png)